docs/user-guide/station-configuration.md

```diff
@@ -4,7 +4,15 @@ Following the LoRaWAN  Network Server (LNS) specification, each Basics Station (
 
 In the following we describe how to register an LBS in IoT Hub and how to store its configuration.
 
-1. Create an IoT Hub device that has a name equal to the LBS EUI in hex-representation, e.g. `DCA632FFFEB32FC6`.
+1. Create an IoT Hub device that has a name equal to the LBS EUI in hex-representation, e.g. `DCA632FFFEB32FC6`. If you are running a dev kit on a Linux machine, the EUI will be retrieved from the MAC address of the eth0 interface as follows:
+
+    ```bash
+    cat /sys/class/net/eth0/address # prints the MAC Address of eth0
+    # Assuming aa:bb:cc:dd:ee:ff is the returned MAC Address
+    # your EUI will be AABBCCFFFEDDEEFF 
+    # Please note the insertion of FFFE in the middle, as per https://doc.sm.tc/station/glossary.html?highlight=mac
```

Comment:
```suggestion
    # Please note the insertion of the literals 'FFFE' in the middle, as per https://doc.sm.tc/station/glossary.html?highlight=mac
```
it's a bit missleading that you have ee:ff in the original address. I would suggest using  gg:hh to differentiate